---
layout: page
title: Papers 
---
**<sup>#</sup>** co-first author,  **\*** co-corresponding author

Jiang C, Trudeau SJ, Cheong TC, Guo R, **Teng M**, Wang LW, Wang Z, Pighi C, 
Gautier-Courteille C, Ma Y, Jiang S, Wang C, Zhao B, Paillard L,
Doench JG, Chiarle R, Gewurz BE. [CRISPR/Cas9 Screens Reveal Multiple Layers 
of B cell CD40 Regulation](https://doi.org/10.1016/j.celrep.2019.06.079).
*Cell Reports*, 2019;28(5):1307-1322.e8.

Wang C, Jiang S, Zhang L, Li D, Liang J, Narita Y, Hou I, Zhong Q,
Gewurz BE, **Teng M\***, Zhao B\*. [TAF family proteins and MEF2C are 
essential for Epstein-Barr virus MYC super-enhancer 
activity](https://doi.org/10.1128/JVI.00513-19).
*Journal of Virology*, 2019;93:e00513-19.

Korthauer K, Kimes PK, Duvallet C, Reyes A, Subramanian A, **Teng M**,
Shukla C, Alm EJ, Hicks SC. [A practical guide to methods controlling 
false discoveries in computational biology](https://doi.org/10.1186/s13059-019-1716-1).
*Genome Biology*, 2019;20:118.

Wang C, Jiang S, Ke L, Zhang L, Li D, Liang J, Narita Y, Hou I, 
Chen CH, Wang LW, Zhong Q, Ling Y, Lv X, Xiang Y, Guo X, **Teng M**, 
Tsao SW, Gewurz BE, Zeng MS, Zhao B. [Genome-wide CRISPR-based
gene knockout screens reveal cellular factors and pathways
essential for nasopharyngeal carcinoma](https://doi.org/10.1074/jbc.RA119.008793).
*Journal of Biological Chemistry*, 2019;294(25):9734-9745.

Wang C, Li D, Zhang L, Jiang S, Liang J, Narita Y, Hou I, Zhong Q, Zheng Z, 
Xiao H, Gewurz BE, **Teng M\***, Zhao B\*. [RNA sequencing analyses of gene
expression during Epstein-Barr virus infection of primary B 
lymphocytes](https://doi.org/10.1128/JVI.00226-19).
*Journal of Virology*, 2019;93:e00226-19.

Chu Y<sup>#</sup>, **Teng M**<sup>#</sup>, Wang Y.
[Modeling and correct the GC bias of tumor and normal WGS data for SCNA
based tumor subclonal population inferring](https://doi.org/10.1186/s12859-018-2099-0).
*BMC Bioinformatics*, 2018;19(Suppl 5):112.

Hicks SC, Townes FW, **Teng M**, Irizarry RA. [Missing data and technical 
variability in single-cell RNA-sequencing experiments](https://doi.org/10.1093/biostatistics/kxx053). 
*Biostatistics*, 2018;19(4):562-578.

**Teng M**, Irizarry RA. [Accounting for GC-content bias reduces systematic 
errors and batch effects in ChIP-seq data](https://doi.org/10.1101/gr.220673.117).
*Genome Research*, 2017;27(11):1930-1938.

Nakayama RT, Pulice JL, Valencia AM, McBride MJ, McKenzie ZM, Gillespie MA, 
Ku WL, **Teng M**, Cui K, Williams RT, Cassel SH, Qing H, Widmer CJ, Demetri GD,
Irizarry RA, Zhao K, Ranish JA, Kadoch C. [SMARCB1 is required for widespread 
BAF complex-mediated activation of enhancers and bivalent promoters](https://doi.org/10.1038/ng.3958).
*Nature Genetics*, 2017;49(11):1613-1623.

Chu Y, Wang Z, Wang R, Zhang N, Li J, Hu Y, **Teng M**, Wang Y. 
[WDNfinder: a method for minimum driver node set detection and analysis in 
directed and weighted biological network](https://doi.org/10.1142/S0219720017500214).
*Journal of Bioinformatics and Computational Biology*, 2017;15(5):1750021.

**Teng M**, Love MI, Davis CA, Djebali S, Dobin A, Graveley BR, Li S, Mason CE,
Olson S, Pervouchine D, Sloan CA, Wei X, Zhan L, Irizarry RA.
[A benchmark for RNA-seq quantification pipelines](https://doi.org/10.1186/s13059-016-0940-1).
*Genome Biology*, 2016;17:74.

Liu B, Guan D, **Teng M**, Wang Y. [rHAT: fast alignment of noisy long reads 
with regional hashing](https://doi.org/10.1093/bioinformatics/btv662).
*Bioinformatics*, 2016;32(11):1625-1631.

Juan L, Liu Y, Wang Y, **Teng M**, Zang T, Wang Y. [Family genome browser: 
visualizing genomes with  pedigree information](https://doi.org/10.1093/bioinformatics/btv151).
*Bioinformatics*, 2015;31(14):2262-2268.

Juan L, **Teng M**, Zang T, Hao Y, Wang Z, Yan C, Liu Y, Li J, Zhang T, Wang Y. 
[The personal genome browser: visualizing functions of genetic variants](https://doi.org/10.1093/nar/gku361).
*Nucleic Acids Research*, 2014;42(W1):W192-W197.

Wang G, Qi K, Zhao Y, Li Y, Juan L, **Teng M**, Li L, Liu Y, Wang Y. 
[Identification of regulatory regions of bidirectional
genes in cervical cancer](https://doi.org/10.1186/1755-8794-6-S1-S5).
*BMC Medical Genomics*, 2013;6(Suppl 1):S5.

**Teng M**, Wang Y, Kim S, Li L, Shen C, Wang G, Liu Y, Huang THM, Nephew KP,
Balch C. [Empirical bayes model comparisons for differential methylation 
analysis](https://doi.org/10.1155/2012/376706).
*Comparative and Functional Genomics*, 2012;2012:376706.

**Teng M**<sup>#</sup>, Ichikawa S<sup>#</sup>, Padgett LR, Wang Y, Mort M, 
Cooper DN, Koller DL, Foroud T, Edenberg HJ, Econs MJ, Liu Y. 
[regSNPs: a strategy for prioritizing regulatory single nucleotide 
substitutions](https://doi.org/10.1093/bioinformatics/bts275).
*Bioinformatics*, 2012;28(14):1879-1886.

**Teng M**<sup>#</sup>, Balch C<sup>#</sup>, Liu Y, Li M, Huang THM, Wang Y,
Nephew KP, Li L. [The influence of cis-regulatory elements on DNA methylation
fidelity](https://doi.org/10.1371/journal.pone.0032928).
*PLoS ONE*, 2012;7(3):e32928.

Zhu S, Jiang Q, Wang G, Liu B, **Teng M**, Wang Y. [Chromatin structure 
characteristics of pre-miRNA genomic sequences](https://doi.org/10.1186/1471-2164-12-329). 
*BMC Genomics*, 2011;12:329.

**Teng M**, Wang Y, Wang G, Jung J, Edenberg HJ, Sanford JR, Liu Y. 
[Prioritizing single-nucleotide variations that potentially regulate alternative
splicing](https://doi.org/10.1186/1753-6561-5-S9-S40).
*BMC Proceedings*, 2011;5(Suppl 9):S40.

Shen C, Huang Y, Liu Y, Wang G, Zhao Y, Wang Z, **Teng M**, Wang Y, Flockhart DA,
Skaar TC, Yan P, Nephew KP, Huang THM, Li L. [A modulated empirical Bayes model for
identifying topological and temporal estrogen receptor α regulatory networks in
breast cancer](https://doi.org/10.1186/1752-0509-5-67).
*BMC Systems Biology*, 2011;5:67.

Zhou FC, Balaraman Y, **Teng M**, Liu Y, Singh RP, Nephew KP. 
[Alcohol alters DNA methylation patterns and inhibits neural stem cell 
differentiation](https://doi.org/10.1111/j.1530-0277.2010.01391.x).
*Alcoholism: Clinical and Experimental Research*, 2011;35(4):735-746.

Wang G, Wang Y, **Teng M**, Zhang D, Li L, Liu Y. [Signal transducers and
activators of transcription-1 (STAT1) regulates microRNA transcription in 
interferon γ-stimulated HeLa cells](https://doi.org/10.1371/journal.pone.0011794).
*PLoS ONE*, 2010;5(7):e11794. 

Jiang Q, Hao Y, Wang G, Juan L, Zhang T, **Teng M**, Liu Y, Wang Y.
[Prioritization of disease microRNAs through a human phenome-microRNAome 
network](https://doi.org/10.1186/1752-0509-4-S1-S2).
*BMC Systems Biology*, 2010;4(Suppl 1):S2.

Jiang Q, Wang Y, Hao Y, Juan L, **Teng M**, Zhang X, Li M, Wang G, Liu Y. 
[miR2Disease: a manually curated database for microRNA deregulation in human
disease](https://doi.org/10.1093/nar/gkn714). *Nucleic Acids Research*, 
2009;37(suppl 1):D98-D104.
