---
layout: page
title: Selected Publications
---

Full list of publications on [Google Scholar](https://scholar.google.com/citations?user=T7QIObwAAAAJ)
or [PubMed](https://pubmed.ncbi.nlm.nih.gov/?term=Teng%2C+Mingxiang%5BAuthor%5D&sort=date&size=20)

<br>

- Guo R, Zhang Y, **Teng M**, ..., Cesarman E, Giulino-Roth L, Gewurz BE. 
(2020) DNA methylation enzymes and PRC1 restrict B-cell Epstein–Barr virus oncoprotein expression.
[*Nature Microbiology*](https://doi.org/10.1038/s41564-020-0724-y)
| [PubMed](https://pubmed.ncbi.nlm.nih.gov/32424339)

- Guo R, Jiang C, Zhang Y, ...,  Doench JG, **Teng M**<sup>‡</sup>, Gewurz BE<sup>‡</sup>.
(2020) MYC controls the Epstein-Barr Virus lytic switch. 
[*Molecular Cell*](https://doi.org/10.1016/j.molcel.2020.03.025)
| [PubMed](https://pubmed.ncbi.nlm.nih.gov/32315601)
(Highlighted in [*Cancer Discovery*](https://doi.org/10.1158/2159-8290.CD-RW2020-064))

- Wang C, Jiang S, Zhang L, ..., Gewurz BE, **Teng M**<sup>‡</sup>, Zhao B<sup>‡</sup>.
(2019) TAF family proteins and MEF2C are essential for Epstein-Barr virus MYC super-enhancer activity. 
[*Journal of Virology*](https://doi.org/10.1128/JVI.00513-19)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31167905)

- Wang C, Li D, Zhang L, ..., Gewurz BE, **Teng M**<sup>‡</sup>, Zhao B<sup>‡</sup>.
(2019) RNA sequencing analyses of gene expression during Epstein-Barr virus infection of primary B lymphocytes.
[*Journal of Virology*](https://doi.org/10.1128/JVI.00226-19)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31019051)

- Hicks SC, Townes FW, **Teng M**, Irizarry RA.
(2018) Missing data and technical variability in single-cell RNA-sequencing experiments.
[*Biostatistics*](https://doi.org/10.1093/biostatistics/kxx053)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29121214)

- Chu Y<sup>†</sup>, **Teng M**<sup>†</sup>, Wang Y. 
(2018) Modeling and correct the GC bias of tumor and normal WGS data for SCNA based tumor subclonal population inferring.
[*BMC Bioinformatics*](https://doi.org/10.1186/s12859-018-2099-0) 
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29671389)

- **Teng M**, Irizarry RA. 
(2017) Accounting for GC-content bias reduces systematic errors and batch effects in ChIP-seq data.
[*Genome Research*](https://doi.org/10.1101/gr.220673.117)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29025895)

- **Teng M**, Love MI, Davis CA, ..., Wei X, Zhan L, Irizarry RA.
(2016) A benchmark for RNA-seq quantification pipelines.
[*Genome Biology*](https://doi.org/10.1186/s13059-016-0940-1)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27107712)
(Highlighted in [*Nature Reviews Genetics*](https://www.nature.com/articles/nrg.2016.62))

- Juan L, **Teng M**, Zang T, ..., Li J, Zhang T, Wang Y.
(2014) The personal genome browser: visualizing functions of genetic variants.
[*Nucleic Acids Research*](https://doi.org/10.1093/nar/gku361)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/24799434)

- **Teng M**<sup>†</sup>, Ichikawa S<sup>†</sup>, Padgett LR, ..., Edenberg HJ, Econs MJ, Liu Y.
(2012) regSNPs: a strategy for prioritizing regulatory single nucleotide substitutions.
[*Bioinformatics*](https://doi.org/10.1093/bioinformatics/bts275)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22611130)

- **Teng M**<sup>†</sup>, Balch C<sup>†</sup>, Liu Y, ..., Wang Y, Nephew KP, Li L.
(2012) The influence of cis-regulatory elements on DNA methylation fidelity.
[*PLoS ONE*](https://doi.org/10.1371/journal.pone.0032928)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22412954)