---
layout: page
title: Publications
---

#### 2020

- Vena F, Bayle S, Nieto A, Quereda V, Aceti M, Frydman S, Sansil S, Grant W, 
Monastyrskyi A, McDonald P, Roush W, **Teng M**, Duckett D. (2020) Targeting casein 
kinase 1 delta sensitizes pancreatic and bladder cancer cells to gemcitabine 
treatment by upregulating deoxycytidine kinase. *Mol Cancer Ther* (in press)

- Guo R, Zhang Y, **Teng M**, Jiang C, Schineller M, Zhao B, Doench JG, O’Reilly RJ, 
Cesarman E, Guilino-Roth L, Gewurz BE. (2020) DNA Methylation Enzymes and PRC1 Restrict
B-cell Epstein-Barr Virus Oncoprotein Expression. 
*Nat Microbiol* (in press)

- Velde RV, Yoon N, Marusyk V, Durmaz A, Dhawan A, Myroshnychenko D, Lozano-Peral
D, Desai B, Balynska O, Poleszhuk J, Kenian L, **Teng M**, Abazeed M, Mian O, 
Tan AC, Haura E, Scott J, Marusyk A. (2020) Resistance to targeted therapies as a 
multifactorial, gradual adaptation to inhibitor specific selective pressures.
*Nat Commun* (in press)

- Guo R, Jiang C, Zhang Y, Govande A, Trudeau SJ, Chen F, Fry CJ, Puri R, 
Wolinsky E, Schineller M, Frost TC, Gebre M, Zhao B, Giulino-Roth L, Doench JG,
**Teng M<sup>‡</sup>**, Gewurz BE**<sup>‡</sup>**. (2020) MYC controls the Epstein-Barr 
Virus lytic switch. [*Mol Cell* 78:1-17](https://doi.org/10.1016/j.molcel.2020.03.025)

<br>
#### 2019

- Jiang C, Trudeau SJ, Cheong TC, Guo R, **Teng M**, Wang LW, Wang Z, Pighi C, 
Gautier-Courteille C, Ma Y, Jiang S, Wang C, Zhao B, Paillard L, Doench JG, 
Chiarle R, Gewurz BE. (2019) CRISPR/Cas9 Screens Reveal Multiple Layers of B cell CD40
Regulation. [*Cell Rep* 28(5):1307-22](https://doi.org/10.1016/j.celrep.2019.06.079)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31365872)

- Korthauer K, Kimes PK, Duvallet C, Reyes A, Subramanian A, **Teng M**, Shukla C,
Alm EJ, Hicks SC. (2019) A practical guide to methods controlling false discoveries in
computational biology. [*Genome Biol* 20:118](https://doi.org/10.1186/s13059-019-1716-1)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31164141)

- Wang C, Jiang S, Zhang L, Li D, Liang J, Narita Y, Hou I, Zhong Q, Gewurz BE,
**Teng M<sup>‡</sup>**, Zhao B**<sup>‡</sup>**. (2019) TAF family proteins and MEF2C are 
essential for Epstein-Barr virus MYC super-enhancer activity. [*J Virol* 
93:e00513-19](https://doi.org/10.1128/JVI.00513-19)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31167905)

- Wang C, Li D, Zhang L, Jiang S, Liang J, Narita Y, Hou I, Zhong Q, Zheng Z, Xiao
H, Gewurz BE, **Teng M<sup>‡</sup>**, Zhao B**<sup>‡</sup>**. (2019) RNA sequencing analyses
of gene expression during Epstein-Barr virus infection of primary B lymphocytes.
[*J Virol* 93:e00226-19](https://doi.org/10.1128/JVI.00226-19)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31019051)

- Wang C, Jiang S, Ke L, Zhang L, Li D, Liang J, Narita Y, Hou I, Chen CH, Wang
LW, Zhong Q, Ling Y, Lv X, Xiang Y, Guo X, **Teng M**, Tsao SW, Gewurz BE, Zeng
MS, Zhao B. (2019) Genome-wide CRISPR-based gene knockout screens reveal cellular 
factors and pathways essential for nasopharyngeal carcinoma. [*J Biol Chem* 
294(25):9734-45](https://doi.org/10.1074/jbc.RA119.008793)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31073033)

<br>
#### 2018 Before

- Hicks SC, Townes FW, **Teng M**, Irizarry RA. (2018) Missing data and technical 
variability in single-cell RNA-sequencing experiments. [*Biostatistics* 
19(4):562-8](https://doi.org/10.1093/biostatistics/kxx053)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29121214)

- Chu Y**<sup>†</sup>**, **Teng M<sup>†</sup>**, Wang Y. (2018) Modeling and correct the GC 
bias of tumor and normal WGS data for SCNA based tumor subclonal population 
inferring. [*BMC Bioinformatics* 19(S5):112](https://doi.org/10.1186/s12859-018-2099-0)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29671389)

- **Teng M**, Irizarry RA. (2017) Accounting for GC-content bias reduces systematic errors
and batch effects in ChIP-seq data. [*Genome Res* 27(11):1930-8](https://doi.org/10.1101/gr.220673.117)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29025895)

- Nakayama RT, Pulice JL, Valencia AM, McBride MJ, McKenzie ZM, Gillespie MA, Ku
WL, **Teng M**, Cui K, Williams RT, Cassel SH, Qing H, Widmer CJ, Demetri GD,
Irizarry RA, Zhao K, Ranish JA, Kadoch C. (2017) SMARCB1 is required for widespread 
BAF complex-mediated activation of enhancers and bivalent promoters.
[*Nat Genet* 49(11):1613-23](https://doi.org/10.1038/ng.3958)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28945250)

- Chu Y, Wang Z, Wang R, Zhang N, Li J, Hu Y, **Teng M**, Wang Y. (2017) WDNfinder: a 
method for minimum driver node set detection and analysis in directed and 
weighted biological network. [*J Bioinf Comput Biol* 15(5):1750021](https://doi.org/10.1142/S0219720017500214)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28918707)

- **Teng M**, Love MI, Davis CA, Djebali S, Dobin A, Graveley BR, Li S, Mason CE,
Olson S, Pervouchine D, Sloan CA, Wei X, Zhan L, Irizarry RA. (2016) A benchmark for
RNA-seq quantification pipelines. [*Genome Biol* 17:74](https://doi.org/10.1186/s13059-016-0940-1)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27107712)

- Liu B, Guan D, **Teng M**, Wang Y. (2016) rHAT: fast alignment of noisy long reads 
with regional hashing. [*Bioinformatics* 32(11):1625-31](https://doi.org/10.1093/bioinformatics/btv662)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26568628)

- Juan L, Liu Y, Wang Y, **Teng M**, Zang T, Wang Y. (2015) Family genome browser: 
visualizing genomes with pedigree information. [*Bioinformatics* 31(14):2262-8](https://doi.org/10.1093/bioinformatics/btv151) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25788626)

- Juan L, **Teng M**, Zang T, Hao Y, Wang Z, Yan C, Liu Y, Li J, Zhang T, Wang Y. (2014)
The personal genome browser: visualizing functions of genetic variants.
[*Nucleic Acids Res* 42(W1):W192-7](https://doi.org/10.1093/nar/gku361)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/24799434)

- Wang G, Qi K, Zhao Y, Li Y, Juan L, **Teng M**, Li L, Liu Y, Wang Y. 
(2013) Identification of regulatory regions of bidirectional genes in cervical cancer.
[*BMC Med Genomics* 6(S1):S5](https://doi.org/10.1186/1755-8794-6-S1-S5)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23369456)

- **Teng M**, Wang Y, Kim S, Li L, Shen C, Wang G, Liu Y, Huang THM, Nephew KP,
Balch C. (2012) Empirical bayes model comparisons for differential methylation analysis.
[*Comp Funct Genom* 2012:376706](https://doi.org/10.1155/2012/376706)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22956892)

- **Teng M<sup>†</sup>**, Ichikawa S**<sup>†</sup>**, Padgett LR, Wang Y, Mort M, 
Cooper DN, Koller DL, Foroud T, Edenberg HJ, Econs MJ, Liu Y. (2012) regSNPs: a 
strategy for prioritizing regulatory single nucleotide substitutions.
[*Bioinformatics* 28(14):1879-86](https://doi.org/10.1093/bioinformatics/bts275)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22611130)

- **Teng M<sup>†</sup>**, Balch C**<sup>†</sup>**, Liu Y, Li M, Huang THM, Wang Y,
Nephew KP, Li L. (2012) The influence of cis-regulatory elements on DNA methylation
fidelity. [*PLoS ONE* 7(3):e32928](https://doi.org/10.1371/journal.pone.0032928)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22412954)

- Zhu S, Jiang Q, Wang G, Liu B, **Teng M**, Wang Y. (2011) Chromatin structure 
characteristics of pre-miRNA genomic sequences. [*BMC Genomics* 12:329](https://doi.org/10.1186/1471-2164-12-329)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21702984)

- **Teng M**, Wang Y, Wang G, Jung J, Edenberg HJ, Sanford JR, Liu Y. 
(2011) Prioritizing single-nucleotide variations that potentially regulate alternative
splicing. [*BMC Proc* 5(S9):S40](https://doi.org/10.1186/1753-6561-5-S9-S40)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22373210)

- Shen C, Huang Y, Liu Y, Wang G, Zhao Y, Wang Z, **Teng M**, Wang Y, Flockhart DA,
Skaar TC, Yan P, Nephew KP, Huang THM, Li L. (2011) A modulated empirical Bayes model for
identifying topological and temporal estrogen receptor α regulatory networks in
breast cancer. [*BMC Syst Biol* 5:67](https://doi.org/10.1186/1752-0509-5-67)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21554733)

- Zhou FC, Balaraman Y, **Teng M**, Liu Y, Singh RP, Nephew KP. (2011) Alcohol alters DNA
methylation patterns and inhibits neural stem cell differentiation. [*Alcohol Clin
Exp Res* 35(4):735-46](https://doi.org/10.1111/j.1530-0277.2010.01391.x)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21223309)

- Wang G, Wang Y, **Teng M**, Zhang D, Li L, Liu Y. (2010) Signal transducers and
activators of transcription-1 (STAT1) regulates microRNA transcription in 
interferon γ-stimulated HeLa cells. [*PLoS ONE* 5(7):e11794](https://doi.org/10.1371/journal.pone.0011794)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20668688)

- Jiang Q, Hao Y, Wang G, Juan L, Zhang T, **Teng M**, Liu Y, Wang Y. (2010) Prioritization
of disease microRNAs through a human phenome-microRNAome network. [*BMC Syst Biol*
4(S1):S2](https://doi.org/10.1186/1752-0509-4-S1-S2)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20522252)

- Jiang Q, Wang Y, Hao Y, Juan L, **Teng M**, Zhang X, Li M, Wang G, Liu Y. (2009)
miR2Disease: a manually curated database for microRNA deregulation in human disease.
[*Nucleic Acids Res* 37(S1):D98-104](https://doi.org/10.1093/nar/gkn714)
| [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/18927107)
