---
layout: page
title: Research Interests
---

We focus on data challenges in computational epigenomics, including data 
harmonization, data integration and functional annotation. Two main research areas
are listed below. <br>

---

#### Integrative Epigenomics 

With epigenomics dataset accumulated, opportunities emerge to better interpret gene 
regulation using integrative approaches. A variety of mature technologies dedicate
to producing genome-wide profiles of epigenomic data: ChIP-seq, CUT&Tag, ATAC-seq, 
DNase-seq, HiC, HiChIP, and CRISPR screens etc. Our lab develop computational methods
to process and understand these datasets.

 - **Harmonizing ChIP-seq datasets across large spectrum of biological conditions**.
   One barrier to jointly interpret ChIP-seq is the [data  heterogeneity](https://doi.org/10.1093/nargab/lqab098) 
   that raise from diverse experimental parameters in different studies. For example,
   different ChIP-seq samples might be tolerant to different levels of PCR artifacts 
   that are associated to the [GC content of sequencing fragments](https://doi.org/10.1101/gr.220673.117).
   We have introduced statistical models to deconvolute such data heterogeneity and
   better interpret ChIP-seq signals.

 - **Understanding enhancer function with multi-omics integration**. Key questions to
   interpret enhancer function include who and how enhancers organize gene regulation.
   Sequencing of 3D chromatin looping has provided precise measurements of enhancer-gene
   relationships. When combined with other sequencing data, it facilitates the [enhancer
   regulator identification](https://doi.org/10.1038/s41467-020-20136-w) in cancers,
   and provides new insights to the [enhancer dynamic characteristics](https://doi.org/10.1093/nar/gkac141)
   in cell-type-specific gene regulation.

 - **Standardizing gene expression to aid epigenomic interpretation**. Gene expression
   is the most-widely used metric to evaluate and validate epigenomic effects. In studies 
   where multiple samples/conditions/parameters are involved, [benchmarked gene 
   quantification](https://doi.org/10.1186/s13059-016-0940-1) is critical so that 
   downstream comparison and evaluation is fair and accurate. Specially, sometimes it is
   important to organize the [dynamic patterns of gene expression](https://doi.org/10.1128/JVI.00226-19)
   across conditions in order to extract the epigenomic effects
   efficiently.
   <br>

---

#### Cancer Genomics

Cancers are the main biological setting where we apply our computational techniques. Our
recent works focus on understanding oncogenesis mechanisms based on close collaboration
with web-lab colleagues. We study oncogenesis mechanisms triggered by oncoviruses like
Epsteinâ€“Barr virus (EBV) and human papillomavirus (HPV), and other reasons etc.

 - **Oncogenesis driven by viral-associated 3D chromatin looping**. One of our key hypotheses
   is that viruses alter the host genome 3D looping during oncogenesis. This provides us an
   unique angle to identify [key biomarkers in viral-triggered cancers](https://doi.org/10.1038/s41467-020-20136-w)
   Particularly, [super enhancers](https://doi.org/10.1128/JVI.00513-19) might be heavily 
   involved in such oncogenesis processes. 

 - **Cancer biomarker identification driven by CRISPR screens**. CRIPSR screens have been
   successfully applied to nominate driver biomarkers in cancers, especially with the aid of
   other sequencing datasets and validation experiments. For example, we detected new roles 
   of [MYC gene in EBV-associated cancers](https://doi.org/10.1016/j.molcel.2020.03.025)
   through CRISPR screens.  
