---
layout: page
title: Research Interests
---

Our research focuses on data science questions in decoding cancer
epigenomic mechanisms. Our approaches include computational/statistical
modeling, large data integration, and close wet-lab collaboration.
Main projects are listed below. <br>

---

#### Integrative Epigenomics 

Accumulated datasets in epigenomics enable us to better
interpret gene regulation through integrative approaches. These
datasets include that measuring gene expression (RNA-seq, scRNA-seq
etc.), chromatin protein binding (ChIP-seq,
CUT&Tag etc.), chromatin accessibility (ATAC-seq, DNase-seq,
scATAC-seq etc.),
3D chromatin looping (HiC, HiChIP etc.), genome
editing in non-coding regions (CRISPR screens etc.) and so on. Our lab
develop computational methods to integrate and interpret these datasets.

 - **Understanding enhancer function with multi-omics data
   integration**. Key questions to interpret enhancer function include
   who and how enhancers organize gene regulation. We integrate
   sequencing assays measuring enhancer acitivities, enhancer-protein
   binding, gene expression and enhaner-gene interactions to provide
   quantitative answers to the questions. With well-designed
   integrative approaches, we uncovered novel insights to understand [super
   enhancer internal
   organization](https://doi.org/10.1093/nar/gkac141) and their roles
   in [defining cancer identities](https://doi.org/10.1371/journal.pcbi.1011873). 
   By integrating public datasets with in-house validations, we
   identified [critical enhancer regulators](https://doi.org/10.1038/s41467-020-20136-w) in cancers.

- **Harmonizing epigenomic sequencing variabilities across wide
   biological conditions**. One barrier to efficiently integrate
   epigenomic sequencing datasets is the data heterogeneities within one
   data modality and across multi-modalities. The heterogeneities
   raise from diverse biological and technical parameters in different
   studies. For example, ChIP-seq datasets are tolerant to high
   measuring variabilities due to [PCR induced GC content
   biases](https://doi.org/10.1101/gr.220673.117) and other [intrisic
   bias factors](https://doi.org/10.1093/nargab/lqab098). We develop
   statistical models to deconvolute such data heterogeneity and
   improve the interpretation of epigenomic sequencing signals.
   <br>

---

#### Cancer Genomics

Cancers are the main biological settings where we apply our
computational methodologies . Our recent work focuses on understanding
oncogenesis mechanisms based on close collaboration
with investigators from diverse background. We study oncogenesis mechanisms triggered by
oncogenic viruses (e.g. Epsteinâ€“Barr virus and human papillomavirus etc.) and clonal hematopoiesis. 

 - **Oncogenesis driven by virus-triggered 3D chromatin
   looping**. One of our key hypotheses is that viruses alter the host
   genome 3D looping during infection in diferent cancers. This provides us an
   unique angle to evaluate [novel gene biomarkers in virus-triggered
   cancers](https://doi.org/10.1038/s41467-023-37347-6). We also found that
   viruses re-organize [critical super
   enhancers](https://doi.org/10.1038/s41467-020-20136-w) 
   in re-wiring tumor genome expression. 

 - **Clonal hematopoiesis biomarkers across solid tumors**. Clonal
   hematopoiesis was found highly associated to reduced survival in
   cancer patients. The molecular mechanisms however are unclear. We work
   closely with the ORIEN network to decode relationships between
   cancers and clonal hematopoiesis, to understand the common and
   unique [roles of clonal hematopoiesis](https://doi.org/10.1093/infdis/jiae212).  
