---
layout: page
title: Tools
---

#### Maintainer

[gcapc](https://bioconductor.org/packages/gcapc/).
An R/Bioconductor package for ChIP-seq peak calling with adjustment of
potential GC-content bias. Also available
[here](https://github.com/tengmx/gcapc) on GitHub.

[ComplexDiff](https://github.com/tengmx/ComplexDiff).
An R-package to normalize and estimate differential DNA bindings
of protein complexes with ChIP-seq data.

[rnaseqcomp](https://bioconductor.org/packages/rnaseqcomp).
An R/Bioconductor pakcage providing several quantitative and visualized
benchmarks for RNA-seq quantification pipelines, accompanied with an
easy-to-use [web tool](http://rafalab.rc.fas.harvard.edu/rnaseqbenchmark).
Also available [here](https://github.com/tengmx/rnaseqcomp) on GitHub.

[regSNPs](https://github.com/tengmx/regSNPs).
R code implementing the strategy for prioritizing regulatory 
single nucleotide substitutions.

<br>
#### Contributor

[WDNfinder](https://github.com/dustincys/WDNfinder).
A method for detection of minimum driver node set.

[Pre-SCNAClonal](https://github.com/dustincys/Pre-SCNAClonal).
Tumor subclonal population inferring tool.

[FGB](http://mlg.hit.edu.cn/FGB/).
A genome browser that provides comprehensive analysis and 
visualization for family genomes.

[rHAT](https://github.com/HIT-Bioinformatics/rHAT).
A sequence alignment tool that provides fast alignment for 
noisy long reads with regional hashing.

[PGB](http://www.pgbrowser.org).
A genome browser that provides comprehensive functional 
annotation and visualization for human individual genomes.

[miR2Disease](http://www.miR2Disease.org).
A manually curated database for microRNA deregulation in human disease.
